Dockerfile

```diff
@@ -1,22 +1,52 @@
-FROM gapsystem/gap-container
+FROM ubuntu:bionic
 
 MAINTAINER The GAP Group <support@gap-system.org>
 
 ENV DEBIAN_FRONTEND noninteractive
 
-# Prerequirements
-RUN    sudo apt-get update -qq \
-    && sudo ln -fs /usr/share/zoneinfo/America/New_York /etc/localtime \
-    && sudo apt-get -qq install -y \
-                                libboost-dev \
-                                libgmp-dev libgmpxx4ldbl \
-                                libmpfr-dev libmpfi-dev libmpc-dev \
-                                autoconf autogen libtool libreadline6-dev libglpk-dev \
-                                libmpfr-dev libcdd-dev libntl-dev git polymake
+# Prerequisites
+RUN    dpkg --add-architecture i386 \
+    && apt-get update -qq \
+    && apt-get -qq install -y \
+            autoconf \
+            autogen \
+            automake \
+            build-essential \
+            cmake \
+            g++ \
+            gcc \
+            gcc-multilib \
+            git \
+            libboost-dev \
+            libcdd-dev \
```

Comment:
@alex-konovalov @sebasguts Any idea why this is here? Best I can think of is that polymake uses cdd; but unless one wants to compile polymake, that shouldn't matter... Can I remove it?

Reply:
Perhaps 4ti2 (which we build from source later on) needs it?

Reply:
I think for polymake, but I am not sure.